tests/unit-tests/admin/test-class-sensei-learners-admin-bulk-actions-view.php

```diff
@@ -0,0 +1,159 @@
+<?php
+require_once SENSEI_TEST_FRAMEWORK_DIR . '/trait-sensei-course-enrolment-test-helpers.php';
+
+/**
+ * Sensei Learners Admin Bulk Actions View Class Unit Tests
+ *
+ * @covers Sensei_Learners_Admin_Bulk_Actions_View
+ */
+class Sensei_Learners_Admin_Bulk_Actions_View_Test extends WP_UnitTestCase {
+	use Sensei_Course_Enrolment_Test_Helpers;
+
+	/**
+	 * Hook suffix.
+	 *
+	 * @var string|null
+	 */
+	private static $initial_hook_suffix;
+
+	/**
+	 * Factory for setting up testing data.
+	 *
+	 * @var Sensei_Factory
+	 */
+	protected $factory;
+
+	public static function setUpBeforeClass() {
+		parent::setUpBeforeClass();
+		self::$initial_hook_suffix = $GLOBALS['hook_suffix'] ?? null;
+		$GLOBALS['hook_suffix']    = null;
+	}
+
+	public static function tearDownAfterClass() {
+		parent::tearDownAfterClass();
+		self::resetEnrolmentProviders();
+		$GLOBALS['hook_suffix'] = self::$initial_hook_suffix;
+	}
+
+	public function setUp() {
+		parent::setUp();
+		$this->factory = new Sensei_Factory();
+		self::resetEnrolmentProviders();
+	}
+
+	/**
+	 * Tests that the user gets redirected when the action is invalid.
```

Comment:
Not a part of your changes I think :p But feel free to fix the comment

Reply:
Fixed here: https://github.com/Automattic/sensei/pull/5043/commits/76f0b70eb91d2aadcb2d930e23471242f16591ca